web_ir_actions_act_window_message/README.rst

```diff
@@ -14,6 +14,35 @@ Depend on this module and return
         'type': 'ir.actions.act_window.message',
         'title': _('My title'),
         'message': _('My message'),
+        # optional title of the close button, if not set, will be _('Close')
+        # if set False, no close button will be shown
+        # you can create your own close button with an action of type
+        # ir.actions.act_window_close
+        'close_button_title': 'Make this window go away',
+        # this is an optional list of buttons to show
+        'buttons': [
+            # a button can be any action (also ir.actions.report.xml et al)
+            {
+                'type': 'ir.actions.act_window',
+                'name': 'All customers',
+                'res_model': 'res.partner',
+                'view_mode': 'form',
+                'views': [[False, 'list'], [False, 'form']],
+                'domain': [('customer', '=', True)],
+            },
+            # or if type == method, you need to pass a model, a method name and
+            # parameters
+            {
+                'type': 'method',
```

Comment:
I also considered that, but decided against it in the end because this is sufficiently different from view buttons, and I hate having to type `object` for calling a method every single time I put a button on the form ;-)